operations/helm/charts/mimir-distributed/templates/store-gateway/store-gateway-statefulset.yaml

```diff
@@ -57,8 +57,20 @@ spec:
     metadata:
       labels:
         {{- include "mimir.podLabels" $args | nindent 8 }}
+        {{- if (eq $rolloutZone.noDownscale true )}}
+        grafana.com/no-downscale: {{$rolloutZone.noDownscale}}
+        {{- else }}
+        grafana.com/prepare-downscale: {{$rolloutZone.prepareDownscale}}
```

Comment:
do we need the annotation even if prepareDownscale is false? I think we can just not apply it